src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1239,7 +1239,7 @@ public void LangVersion()
             parsedArgs = DefaultParse(new[] { "/langversion:latest", "a.cs" }, _baseDirectory);
             parsedArgs.Errors.Verify();
             Assert.Equal(LanguageVersion.Latest, parsedArgs.ParseOptions.SpecifiedLanguageVersion);
-            Assert.Equal(defaultVersion, parsedArgs.ParseOptions.LanguageVersion);
+            Assert.Equal(LanguageVersion.CSharp7_1, parsedArgs.ParseOptions.LanguageVersion);
```

Comment:
I think we need at least one test validating that we indeed bumped the latest. 

---
In reply to: [101625158](https://github.com/dotnet/roslyn/pull/17169#discussion_r101625158) [](ancestors = 101625158)